README.md

```diff
@@ -1,36 +1,32 @@
-# DJANGO SOCIAL REST API
+# DJANGO ON-DEMAND
 
-Welcome to this open project aimed to help building una buena rest api used for social app.
+Manage suppliers and consumers in an on-demand marketplace. 
+More info about on-demand: https://www.pubnub.com/blog/what-is-the-on-demand-economy/
 
 Yes, fork it.
 
 Please, contribute.
 
 Thanks!
 
-### Instal dependencies
-`pip3 install -r requirements.txt`
+Full documentation on [insert documentation url]
 
-### Install mysqlclient library
-**NOTE:** On some installations Django has issues recognized the installed library. This was solved by installing versions of the library previous to `1.3.13`. For example, installing this specific one `pip3 install mysqlclient==1.3.12` resulted in success.
+### Install app
+This app is still work in progress. To install the work in progress version 
+`pip3 install -i https://test.pypi.org/simple/ django-on-demand`
 
-### MySQL
-Set an empty password for user `root` or change the password on `DATABASES` section on `settings.py` but don't push the changes!
+### INSTALLED_APPS requirement
+This app is using `django-rest-framework` to provide API endpoints, which means this app will need to be added to the list of installed apps.
+The app will automatically install `django-rest-framework` if it's not installed.
+`on_demand` also needs to be added to the list of installed apps.
+```
+ INSTALLED_APPS = [
+        ...
+        'on_demand',
+        'rest_framework'
+    ]
+```
 
-After doing this, create a database called `socialapp`.
+Run `python3 manage.py migrate` to get the models from the app.
 
-Then, located in the root directory (where `manage.py` is) run `python3 manage.py migrate` - This will create all the tables and relationships from the models in the db.
-
-## Config
-Properties that depend on the environment (production vs pre production environment), such as passworks, keys and tokens should be configured in a file named `local_settings.py` under `socialapp` directory.
-
-This file is *gitignored* to ensure this sensitive data is not published on the repository and that values are explicitly set for the environment where the server is running.
-
-### Run the server
-By running `python3 manage.py runserver` you'll be able to start the server.
-
-If you are using VScode, to enable the linter do:
-* Go to Code -> Preferences -> Settings, and add the following to the user settings json:
-    `    "python.linting.pylintArgs": [
-        "--load-plugins=pylint_django"
-   ]`
\ No newline at end of file
+Include the Social Rest API URLconf in your project `urls.py` like this: `path('ondemand/', include('on_demand.urls'))`
```

Comment:
Probably we should change here `social rest api` because can me misleading. We should try to keep referring as on demand api, for the sake of consistency.

Reply:
ah, forgot